.codeclimate.yml

```diff
@@ -13,3 +13,6 @@ checks:
 
 exclude_patterns:
   - "src/serviceWorker.js"
+  - "src/actions/__tests__/"
+  - "src/reducers/__tests__/"
+  - "**/*.spec.jsx"
```

Comment:
great work Joseph, I think this line was enough to cover all test files. no need to include test folders. then maybe you could add "**/*.spec.js" in case we ever need to write tests that do not have JSX in them

Reply:
@bibangamba I think excluding the test folder is better since it covers a wide scope. for example, someone may prefer to name his test files like "test.js" or "test.jsx" we should be refactoring all the time.